modules/claims_api/app/sidekiq/claims_api/va_notify_job.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VANotifyJob < ClaimsApi::ServiceBase
+    def perform(poa_id, rep)
+      return if skip_notification_email?
+
+      poa = ClaimsApi::PowerOfAttorney.find(poa_id)
+      form_data = poa.form_data
```

Comment:
I did not really feel like that was necessary since we would never get this far if there wasn't one.

Reply:
true, and we would log the "nil has no form_data" exception below, making it easy to tell what went wrong. more of a best practice, since we don't know how our job will be used in the future.

Reply:
Sold, [2dfa039](https://github.com/department-of-veterans-affairs/vets-api/pull/18687/commits/2dfa0397f37d611682a67419afd8c1101da6212d)